consensus/Cargo.toml

```diff
@@ -63,6 +69,19 @@ version = "1.0"
 [dependencies.mpmc-map]
 version = "0.1"
 
+[dependencies.indexmap]
+version = "1.6"
+
+[dependencies.lazy_static]
+version = "1.4"
+
+[dependencies.dyn-clone]
+version = "1.0"
+
+[dependencies.snarkos-metrics]
+path = "../metrics"
+version = "1.3.9"
```

Comment:
The current version is `1.3.10`; the PR needs to be made against `staging` instead of `master`, so it probably needs a small rebase too.